src/services/fungibleTokens.js

```diff
@@ -0,0 +1,90 @@
+import * as Sentry from '@sentry/browser';
+import BN from 'bn.js';
+import * as nearApiJs from 'near-api-js';
+
+import sendJson from '../tmp_fetch_send_json';
+import { wallet } from '../utils/wallet';
+import { ACCOUNT_HELPER_URL } from '../utils/wallet';
+
+const FT_MINIMUM_STORAGE_BALANCE = '1250000000000000000000';
+const FT_STORAGE_DEPOSIT_GAS = '30000000000000';
+const FT_TRANSFER_GAS = '30000000000000';
+const FT_TRANSFER_DEPOSIT = '1';
```

Comment:
I took `FT_MINIMUM_STORAGE_BALANCE` and `FT_STORAGE_DEPOSIT_GAS` from here:
https://docs.near.org/docs/api/naj-cookbook#wrap-and-unwrap-near

I'm not sure about `FT_TRANSFER_GAS` - it depends on many things, but unused gas is refunded:
https://docs.near.org/docs/concepts/gas#attach-extra-gas-get-refunded
I set it to the same value as creating an account (`storage_deposit`), so it should be enough, and it's still a very low amount of tokens - but I guess we should double-check it with someone? While testing, the amount of gas used was much much less.

Reply:
1. For `FT_MINIMUM_STORAGE_BALANCE `, please add a comment here with a link to the cookbook so that it's clear what source of truth we used to make this decision later :)
2. For `FT_TRANSFER_GAS` please add a comment indicating that we arbitrarily set this to the same value as we use for creating an account and that the remainder is refunded.  I'm not sure I understand the reference to `storage_deposit`; is that the same?
3. What about `FT_TRANSFER_DEPOSIT`?

Reply:
Done

Reply:
:+1: awesome, looking good